src/navigation/nav2_bt_navigator/src/bt_navigator.cpp

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) 2018 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <string>
+#include <memory>
+#include <exception>
+#include <chrono>
+#include "nav2_bt_navigator/bt_navigator.hpp"
+
+using namespace std::chrono_literals;
+using nav2_tasks::TaskStatus;
+
+namespace nav2_bt_navigator
+{
+
+BtNavigator::BtNavigator(const std::string & name)
+: nav2_tasks::NavigateToPoseTaskServer(name)
+{
+  RCLCPP_INFO(get_logger(), "BtNavigator::BtNavigator");
+  planner_ = std::make_unique<nav2_tasks::ComputePathToPoseTaskClient>("AStarPlanner", this);
```

Comment:
Should "AStarPlanner" be "PlannerClient" and similarly "DwaController" be "FollowPathClient"? Or am I missing something? I don't think the client is tied to a specific implementation of the server, is it?

Reply:
Yes, you're right. These should be interface names rather than specific implementation names (as it is now). I'll change this.